tests/bi_operators/bi_operators_collocation.cpp

```diff
@@ -138,7 +141,7 @@ SCENARIO("A collocation integrator with approximate diagonal elements",
             cnpy::custom::npy_load<double>("tanhsphericaldiffuse_D_collocation.npy");
         for (int i = 0; i < cavity.size(); ++i) {
           for (int j = 0; j < cavity.size(); ++j) {
-            REQUIRE(reference(i, j) == Approx(results(i, j)));
+            REQUIRE(results(i, j) == Approx(results(i, j)));
```

Comment:
Same comment here. This test is checking a matrix against itself, so it will always pass.